rotkehlchen/externalapis/etherscan.py

```diff
@@ -353,6 +353,10 @@ def _query(
             # success, break out of the loop and return result
             return result
 
+        else:  # will only run if while condition fails # noqa: PLW0120
```

Comment:
Hmmm .... now that I think about it the noqa: PLW0120 maybe wrong. Isn't this equivalent to just not putting an else and having it after the loop?